From 1a1f194cfe7ca1d379f32dc9d99933dfb913e3da Mon Sep 17 00:00:00 2001
From: Dan Walkes <danwalkes@trellis-logic.com>
Date: Sun, 28 Oct 2018 15:09:45 -0600
Subject: [PATCH] Override HOSTCC

Force u-boot-fw-utils to build fw_setenv for the target platform by overriding
the value provided for HOSTCC by bitbake.
---
 tools/env/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/env/Makefile b/tools/env/Makefile
index 38ad118..95b28c0 100644
--- a/tools/env/Makefile
+++ b/tools/env/Makefile
@@ -8,7 +8,7 @@
 # fw_printenv is supposed to run on the target system, which means it should be
 # built with cross tools. Although it may look weird, we only replace "HOSTCC"
 # with "CC" here for the maximum code reuse of scripts/Makefile.host.
-HOSTCC = $(CC)
+override HOSTCC = $(CC)
 
 # Compile for a hosted environment on the target
 HOST_EXTRACFLAGS  = $(patsubst -I%,-idirafter%, $(filter -I%, $(UBOOTINCLUDE))) \
